presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaTableHandle.java

```diff
@@ -64,6 +66,8 @@ public KafkaTableHandle(
             @JsonProperty("messageDataFormat") String messageDataFormat,
             @JsonProperty("keyDataSchemaLocation") Optional<String> keyDataSchemaLocation,
             @JsonProperty("messageDataSchemaLocation") Optional<String> messageDataSchemaLocation,
+            @JsonProperty("keySubject") Optional<String> keySubject,
```

Comment:
Can we have it as a `Properties` is `subject` is used only in SchemaRegistry or ?